lighthouse-core/report/templates/report-template.html

```diff
@@ -23,14 +23,16 @@
   <meta name="viewport" content="width=device-width">
   <title>Lighthouse report: {{ url }}</title>
   <style>{{{ css }}}</style>
-  <script>{{{ script }}}</script>
+  {{{ script }}}
 </head>
 <body>
 
-<script>
-  // expose the original results
+<!--
```

Comment:
i feel like it's useful to keep around, but I don't have the strongest usecase just yet.

e.g. what if a report-viewer customer wants to d/l the JSON from a report they are looking at, so they can archive it and compare past results?

but.. I'm OK with nuking it for now and only bringing it back when there's definite need.